client/app/kubernetes/kubernetes.constants.js

```diff
@@ -0,0 +1,17 @@
+angular.module("managerApp")
+    .constant("KUBERNETES", {
+        // For now only one region is available
+        region: "GRA5",
```

Comment:
`regions`, think about the future 😄

Reply:
The region will be given by API in the future (it's the region of the cluster)